### PR TITLE
Fixing restore bug: 578106

### PR DESCRIPTION
**Pull request by raquelsa**
- When a libman.json file fails to parse initially, we get a null manifest leading to exceptions

### CODE REVIEW

File: src/LibraryManager.Vsix/Json/TextviewCreationListener.cs [hunk 1]
```diff
@@ -76,19 +76,23 @@ public void VsTextViewCreated(IVsTextView textViewAdapter)
         {
             var files = new List<FileIdentifier>();
 
-            foreach (ILibraryInstallationState state in manifest.Libraries.Where(l => l.IsValid(out var errors)))
+            if (manifest != null)
             {
-                IEnumerable<FileIdentifier> stateFiles = await GetFilesWithVersionsAsync(state).ConfigureAwait(false);
-
-                foreach (FileIdentifier fileIdentifier in stateFiles)
+                foreach (ILibraryInstallationState state in manifest.Libraries.Where(l => l.IsValid(out var errors)))
```
Comment (jimmylewis) [inline src/LibraryManager.Vsix/Json/TextviewCreationListener.cs]: nit: var?

Comment (jimmylewis) [reply src/LibraryManager.Vsix/Json/TextviewCreationListener.cs]: For that matter, if you're not using errors, just do `out _` instead.

File: src/LibraryManager.Vsix/Json/TextviewCreationListener.cs [hunk 2]
```diff
@@ -76,19 +76,23 @@ public void VsTextViewCreated(IVsTextView textViewAdapter)
...
-                        files.Add(fileIdentifier);
+                        if (!files.Contains(fileIdentifier))
+                        {
+                            files.Add(fileIdentifier);
+                        }
                     }
                 }
             }
 
+
```
Comment (jimmylewis) [inline src/LibraryManager.Vsix/Json/TextviewCreationListener.cs]: nit: remove redundant empty line.

File: src/LibraryManager.Vsix/Json/TextviewCreationListener.cs (hunk 1 above)
Comment (raquelsa) [reply src/LibraryManager.Vsix/Json/TextviewCreationListener.cs]: This code was there before. We have a bug logged to review use of var throughout LibMan code.

**Pull request merged**